src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -2,19 +2,20 @@
   <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="add-notation-dialog">
     <v-card>
       <v-card-title id="dialog-title">
-        <span v-if="administrativeDissolution" id="dialog-title"><strong>{{displayName}}</strong></span>
-        <span v-else-if="putBackOn" id="dialog-title"><strong>Correction - {{displayName}}</strong></span>
+        <span v-if="isTypeAdministrativeDissolution({ name: this.name })"
```

Comment:
Oh no! My fault. This became more complicated than I thought.

Please continue to have the local getter (so that this template code is clean), and in the getter just call the global state getter.

Reply:
Or... using a JS object property shortcut, you could use this in the template instead:

`v-if="istypeAdministrativeDissolution({ name })"`

Reply:
Looks good.